bedrock/security/templates/security/bug-bounty.html

```diff
@@ -28,7 +28,7 @@ <h2 id="general-eligibility">General Eligibility</h2>
     <li>The security bug must be a part of Mozilla’s code, not the code of a third party.  We will pay bounties for vulnerabilities in third-party libraries incorporated into shipped client code or third-party websites utilized by Mozilla.</li>
     <li>You must not have written the buggy code or otherwise been involved in contributing the buggy code to the Mozilla project.</li>
     <li>You must be old enough to be eligible participate in and receive payment from this program in your jurisdiction, or otherwise qualify to receive payment, whether through consent from your parent or guardian or some other way.</li>
-    <li>You must not be an employee, contractor, or otherwise have a business relationship with the Mozilla Foundation or any of its subsidiaries.</li>
+    <li>You must not be an employee, contractor, or otherwise have a business relationship with Mozilla Foundation or any of its subsidiaries.</li>
```

Comment:
similar to Dan's comment above, would it be possible to use Mozilla
```suggestion
    <li>You must not be an employee, contractor, or otherwise have a business relationship with Mozilla or any of its subsidiaries.</li>
```